lib/monitoring/middleware/conjur_exporter.rb

```diff
@@ -0,0 +1,94 @@
+require 'prometheus/client'
+require 'prometheus/client/formats/text'
+
+module Monitoring
+  module Middleware
+    # Exporter is a Rack middleware that provides a sample implementation of a
+    # Prometheus HTTP exposition endpoint.
+    #
+    # By default it will export the state of the global registry and expose it
+    # under `/metrics`. Use the `:registry` and `:path` options to change the
+    # defaults.
+    class ConjurExporter
+      attr_reader :app, :path, :registry
+
+      FORMATS  = [::Prometheus::Client::Formats::Text].freeze
+      FALLBACK = ::Prometheus::Client::Formats::Text
+
+      def initialize(app, options = {})
+        @app = app
+        @path = options[:path] || '/metrics'
+        @port = options[:port]
```

Comment:
We can probably remove anything related to the port option, it feels like an additional toggle that I don't think we'll be using at all.